tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -356,14 +356,14 @@ def generate_temporary_s3_column_list(manifest_file_schema: str) -> str:
 
     Returns: A string representing SQL columns to create.
         Columns required for import but unused by orca will be filled in with `junk` values.
-        For example, 'orca_archive_location text, key_path text, size_in_bytes bigint, last_update timestamptz, etag text, storage_class text, junk7 text, junk8 text, junk9 text, junk10 text, junk11 text, junk12 text, junk13 text, junk14 text'
+        For example, 'orca_archive_location text, key_path text, size_in_bytes text, last_update timestamptz, etag text, storage_class text, junk7 text, junk8 text, junk9 text, junk10 text, junk11 text, junk12 text, junk13 text, junk14 text'
 
     """
     # Keys indicate columns in the s3 inventory csv. Values indicate the corresponding column in orca.reconcile_s3_object
     column_mappings = {
         "Bucket": "orca_archive_location text",
         "Key": "key_path text",
-        "Size": "size_in_bytes bigint",
+        "Size": "size_in_bytes text",  # delete markers have size: ""
```

Comment:
I would keep `size_in_bytes` as `bigint`.

So when we do the initial load into the temp table using `def trigger_csv_load_from_s3_sql()` we should just add the option `FORCE_NULL(Size)` this will interpret the empty string `""` in the Size column as a NULL value for the column. See https://www.postgresql.org/docs/10/sql-copy.html